lessons/8-intermediate-r-2/lesson_script.R

```diff
@@ -0,0 +1,151 @@
+library(dplyr)
+
+### Copy this from Tuesday's script
+
+my_function <-
+  function(data_url = "https://github.com/cct-datascience/repro-data-sci/raw/r-lessons/lessons/7-intermediate-r-1/lesson-data/Biscoe.csv") {
+    island_dat <- read.csv(data_url)
+    
+    
+    island_dat <- island_dat |>
+      na.omit()
+    
+    
+    island_dat_means <- island_dat |>
+      group_by(species, sex) |>
+      summarize(across(ends_with("mm") | ends_with("g"), mean))
+    
+    island_dat_means_imperial <- island_dat_means |>
+      mutate(
+        across(ends_with("mm"), ~ . * 0.03937008, .names = "{.col}_in"),
+        across(ends_with("g"), ~ . * 0.002204623, .names = "{.col}_lb")
```

Comment:
I can't remember, did you cover using `.names` in `across()`?  If not, might be better to simplify this example and not introduce too many new concepts at once.  If this isn't new, then it's a good opportunity to pause and review.

Reply:
Ah, thanks for reviewing the function! We worked through the process of writing this function on Tuesday, so this whole chunk is actually copy-pasting from the script we wrote on Tuesday. It's a good note about how things might be simplified next time, though!